x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/user_actions_get_users.ts

```diff
@@ -431,6 +431,67 @@ export default ({ getService }: FtrProviderContext): void => {
           expect(assignees).to.eql([]);
           expect(unassignedUsers).to.eql([]);
         });
+
+        it('does not throw with imageUrl set to null', async () => {
+          await updateUserProfileAvatar({
+            supertest,
+            req: {
+              // @ts-expect-error: types are not correct
+              initials: null,
+              // @ts-expect-error: types are not correct
+              color: null,
+              imageUrl: null,
+            },
+            headers: superUserHeaders,
+          });
+
+          const postedCase = await createCase(
+            supertestWithoutAuth,
+            getPostCaseRequest(),
+            200,
+            null,
+            superUserHeaders
+          );
+
+          const res = await getCaseUsers({
+            caseId: postedCase.id,
+            supertest,
+          });
+
+          expect(res.participants[0].avatar).to.eql({});
+          expect(res.reporter.avatar).to.eql({});
+        });
+
+        it('does not throw if the avatar data are not a string', async () => {
```

Comment:
@jonathan-buttner The title of the test is misleading. What it should say is `it does not return any avatar data if they are not a string`. Basically, if the data are not the schema we expect (a string) then we do not return them. For us is like it was never there. The UI will never break because we do not return them and can handle `undefined` values. Does it make sense? Do you think we should follow another approach?

Reply:
Ah perfect! sorry I missed that. That sounds good.